src/code/z_en_item00.c

```diff
@@ -206,7 +206,7 @@ void EnItem00_Init(Actor* thisx, GlobalContext* globalCtx) {
             Item_Give(globalCtx, ITEM_RUPEE_PURPLE);
             break;
         case ITEM00_RUPEE_ORANGE:
-            Item_Give(globalCtx, ITEM_RUPEE_ORANGE);
+            Item_Give(globalCtx, ITEM_RUPEE_GOLD);
```

Comment:
Is this right?

Reply:
I'm not entirely sure. 

There are both separate orange and gold rupees that give 200 rupees. Killing the Takkuri in Termina field drops an orange rupee that rewards you with 200 rupees. Yet giving the curiosity shop man Chateau/Big Poe/Gold dust and you get a gold rupee worth 200 as well.

Something similar is in OoT where there are 6 gold rupee chests throughout the world that give 200 rupees, but you can also kill the skull kid which will drop an orange rupee worth 200 rupees as well.

The motivation for choosing gold over orange was because of the similarities between OoT's and MMs gold/orange rupees, so I just copied OoT. The proper way that needs to be done in MM is to document to distinguish between the two. At the moment though, I don't know which would be which in the enum.

I'll also admit that I did not notice the `ORANGE` in `ITEM00_RUPEE_ORANGE` until this comment

Reply:
I think it's one or other in the game text, it just happens that the item00 model doesn't look much like the GI model. And there is one in OoT that's worth 500, that you can get from diving game, IIRC.

Reply:
Which is which? Is GI what you get in the chest/npc and item00 is the freestanding item? Or is the other way around?

Reply:
GI is the "hold over the head" one

Reply:
Also worth noting, when you sell the Chateau/Big Poe/Gold dust, it is referred to as a "Huge Rupee" with no reference to color.

Reply:
Should be that, then. Especially since the higher value ones have inconsistent colours in other games.

Reply:
You mean `ITEM_RUPEE_HUGE`? No complaints

Reply:
Right now we have:
`ITEM00_RUPEE_ORANGE`
`ITEM_RUPEE_GOLD`
`GI_RUPEE_GOLD`
Would you suggest to make all three Huge?
`ITEM00_RUPEE_HUGE`
`ITEM_RUPEE_HUGE`
`GI_RUPEE_HUGE`

Reply:
Yeah, as long as it is consistent I'm good with it.